modules/utxo-lib/src/bitgo/Unspent.ts

```diff
@@ -169,7 +224,7 @@ export function addToPsbt(
   } else {
     const { witnessScript, redeemScript } = createOutputScript2of3(walletKeys.publicKeys, scriptType);
     psbt.updateInput(inputIndex, {
-      bip32Derivation: [signerIndex, cosignerIndex].map((idx) => ({
+      bip32Derivation: [0, 1, 2].map((idx) => ({
```

Comment:
Why do we need all 3 derivation paths here?

Reply:
So that a "standard" PSBT signer that has the 3 xpubs saved can fully validate the script's pubkeys. Doesn't _really_ matter for the input side, but could still be valuable information if there is some funny business going on.